compiler-rt/test/sanitizer_common/TestCases/fopen64_nullptr.c

```diff
@@ -0,0 +1,8 @@
+// Check that fopen64(NULL, "r") is ok.
+
+// REQUIRES: linux
```

Comment:
The interceptors file defines
```
#define SANITIZER_INTERCEPT_FOPEN64 (SI_GLIBC || SI_SOLARIS32)
```
so `// REQUIRES: glibc` might be a safer bet? (and technically solaris32, but no one will notice). Not all Linux builds will have glibc.